examples/supertuxkart/Dockerfile

```diff
@@ -30,19 +30,16 @@ RUN apt update && apt-get install -y build-essential cmake libbluetooth-dev \
 libcurl4-openssl-dev libenet-dev libfreetype6-dev libharfbuzz-dev libfribidi-dev \
 libgl1-mesa-dev libglew-dev libjpeg-dev libogg-dev libopenal-dev libpng-dev \
 libssl-dev libvorbis-dev libxrandr-dev libx11-dev nettle-dev pkg-config zlib1g-dev \
-git subversion
+git subversion wget xz-utils
```

Comment:
Can we remove git and subversion from the list now that we aren't cloning the two repos? Along those lines, I'm wondering if we can significantly trim this list down since we don't need to compile the code any longer.